src/js/components/Navigation/WelcomeAppbar.jsx

```diff
@@ -318,6 +318,7 @@ class WelcomeAppbar extends Component {
                       <IconButton
                         classes={{ root: classes.iconButtonRoot }}
                         id="profileAvatarHeaderBar"
+
```

Comment:
Are you sure you wanted to add a line break here? I recommend running `npm run lint` to make sure there aren't any ESLint warnings on your changes.